README.md

```diff
@@ -126,11 +145,23 @@ Insert Javascript and CSS required for Amber to function. `Required`
 
     subs_filter '</head>' '<script type="text/javascript" src="/amber/js/amber.js"></script><link rel="stylesheet" type="text/css" href="/amber/css/amber.css"></head>';
 
-Display Farsi version of Javascript and CSS 
+Display Javascript and CSS in Farsi
```

Comment:
This block can be removed, since it's been moved to "Optional Configuration" at the bottom of the page